autogpt_platform/backend/.env.default

```
@@ -89,6 +89,7 @@ MEDIA_GCS_BUCKET_NAME=
 # AI/LLM Services
 OPENAI_API_KEY=
 ANTHROPIC_API_KEY=
+GOOGLE_API_KEY=
 GROQ_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GROQ_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)